sickbeard/providers/hdspace.py

```diff
@@ -180,15 +179,15 @@ def login(self):
         login_params = {
             'uid': self.username,
             'pwd': self.password,
-            'page': 'login',
+            'submit': 'Confirm',
         }
 
         response = self.get_url(self.urls['login'], post_data=login_params, returns='text')
         if not response:
             logger.log('Unable to connect to provider', logger.WARNING)
             return False
 
-        if re.search('Password Incorrect', response):
+        if re.search('Automatic log off after 15 minutes inactivity', response):
```

Comment:
We use regex here in all providers. Although, as you said, we could probably just use "in response" instead in most of them. For now I would keep the regex, for a matter of consistency.